api/docs/v1.52.yaml

```
@@ -1688,6 +1688,42 @@ definitions:
           "WorkDir": "/var/lib/docker/overlay2/ef749362d13333e65fc95c572eb525abbe0052e16e086cb64bc3b98ae9aa6d74/work"
         }
 
+  Storage:
+    description: |
+      Information about the storage used for the container's filesystem.
```

I think we can keep it general:

```suggestion
      Information about the storage used by the container.
```

---

👍🏼 applied in https://github.com/moby/moby/compare/325c183f1c8cc94784375fae580a18cc5be0928b..13ea4292afd127e8e257fc1ef91bffebd6a2ab34